packages/graphql/package.json

```diff
@@ -1,29 +1,45 @@
 {
   "name": "@0xintuition/graphql",
-  "version": "0.1.0",
+  "version": "0.0.5",
```

Comment:
do we want to keep the version bump from the previous PR merging in the updated queries?